packages/orangutan/webpack.config.js

```diff
@@ -0,0 +1,241 @@
+    )
+  })
+}
+
+const config = {
+  mode: isProduction ? 'production' : 'development',
+  entry: webpackEntry,
+  output: {
+    path: path.resolve(__dirname, 'dist/'),
+    filename: '[name].[hash].bundle.js',
+    chunkFilename: '[name].[chunkhash].chunk.js',
+    library: 'orangutan',
+    libraryTarget: 'umd',
+  },
+  optimization: {
+    minimize: true,
+    splitChunks: {
+      maxInitialRequests: 6,
+      maxAsyncRequests: 6,
+      minChunks: 1,
+      cacheGroups: {
+        default: false,
+        vendors: false,
+        /*
+         * common chunks
+         */
+        polyfill: {
+          test: module => {
+            return (
+              module.context &&
+              /node_modules\/(babel-polyfill|core-js|regenerator-runtime)/.test(
+                module.context
+              )
+            )
+          },
+          name: 'polyfill',
+          priority: 9,
+          chunks: 'initial',
+        },
+        react: {
+          test: module => {
+            return (
+              module.context &&
+              /node_modules\/(react|history|redux|styled-components)/.test(
+                module.context
+              ) &&
+              !/node_modules\/@twreporter/.test(module.context)
+            )
+          },
+          name: 'react-base',
+          priority: 10,
+          chunks: 'initial',
+        },
+        twreporter: {
+          test: module => {
+            return (
+              module.context &&
+              module.context.includes('node_modules/@twreporter')
+            )
+          },
+          name: 'twreporter-base',
+          priority: 10,
+          chunks: 'initial',
+          reuseExistingChunk: true,
+        },
+        lodash: {
+          test: module => {
+            return (
+              module.context && module.context.includes('node_modules/lodash')
+            )
+          },
+          name: `lodash`,
+          priority: 11,
+          chunks: 'initial',
+          reuseExistingChunk: true,
+        },
+        /*
+         * package's own chunk
+         *
+         * Example: twreporterCore will be a chunk of package1
+         *
+         * twreporterCore: {
+         *   test: module => {
+         *     return (
+         *      module.context && module.context.includes('node_modules/@twreporter/core')
+         *     )
+         *   },
+         *   name: `${packages.package1}/twreporter-core`,
+         *   priority: 11,
+         *   chunks: 'initial',
+         *   reuseExistingChunk: true,
+         * }
+         */
+      },
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            rootMode: 'upward',
+          },
+        },
+      },
+      {
+        test: /\.(jpe?g|png|gif|mp4)$/i,
+        use: {
+          loader: 'url-loader',
+        },
+      },
+      {
+        test: /\.svg$/,
```

Comment:
Are the svg files imported in `.js` already handled by `babel-plugin-inline-react-svg` in `babel-loader`? Is here we intended to use both `babel-plugin-inline-react-svg` and `@svgr/webpack`?

Reply:
This one is for `@twreporter/orangutan`.
However, in `packages/orangutan`, we don't have any `.`jsx` files.
I could remove it for now.
If we need it, we can add it back.